src/linter/xmlTemplate/Parser.ts

```diff
@@ -297,13 +297,13 @@ export default class Parser {
 				this._addNamespace({
 					localName: null,
 					namespace: attrValue,
-				}, this.#nodeStack.length);
+				}, this.#nodeStack.length + 1); // +1 as the currently created node is not yet on the stack
```

Comment:
if the idea is that `level` counts from 1, I'm fine with this change.- Alternatively, if `level` should correspond to the node's position in the `#nodeStack`, `popTag` could just remove `level - 1`(or swap L171 and L172 and adapt the constant in L178).

Reply:
Thanks. That's a way better solution.